entgql/annotation.go

```diff
@@ -163,11 +163,55 @@ func MapsTo(names ...string) Annotation {
 }
 
 // Type returns a type mapping annotation.
+// The Type() annotation is used to map the underlying
+// GraphQL type to the type.
+//
+// To change the GraphQL type for a type:
+//
+//	func (User) Annotations() []schema.Annotation {
+//		return []schema.Annotation{
+//				entgql.Type("MasterUser"),
+//		}
+//	}
+//
+// To change the GraphQL type for a field (rename enum type):
+//
+//	field.Enum("status").
+//		NamedValues(
+//			"InProgress", "IN_PROGRESS",
+//			"Completed", "COMPLETED",
+//		).
+//		Default("IN_PROGRESS").
+//		Annotations(
+//			entgql.Type("TodoStatus"),
+//		)
 func Type(name string) Annotation {
 	return Annotation{Type: name}
 }
 
 // Skip returns a skip annotation.
+// The Skip() annotation is used to skip
+// generating the type or the field from GraphQL schema.
+//
+// It gives you the flexibility to skip generating
+// the type or the field based on the SkipMode flags.
+//
+// For example, if you don't want to expose a field on the <T>WhereInput type
+// you can use the following:
+//
+//	field.String("name").
+//		Annotations(
+//			entgql.Skip(entgql.SkipWhereInput),
+//		)
+//
+// The SkipMode flag is a bit flag, so you can
+// skip multiple modes with the bitwise OR operator:
+// `entgql.SkipWhereInput | entgql.SkipEnumField`
```

Comment:
```suggestion
// Since SkipMode is a bit flag, it's possible to skip multiple modes using
// the bitwise OR operator as follows:
//
//	entgql.Skip(entgql.SkipWhereInput | entgql.SkipEnumField)
//
```